specification/personal-demographics.yaml

```diff
@@ -2096,16 +2096,6 @@ components:
       in: header
       description: |
         The X-Correlation-ID from the request header, if supplied, mirrored back.
-    XRequestID:
-      name: X-Request-ID
-      required: true
-      schema:
-        type: string
-        pattern: '^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$'
-        example: 60E0B220-8136-4CA5-AE46-1D97EF59D068
-      in: header
-      description: |
-        The X-Request-ID from the request header, if supplied, mirrored back.
```

Comment:
This is unused code. The specs actually use the file XRequestID.yaml to build the documentation for the X-Request-ID response header.